app/views/hosts/_dhcp_lease_errors.html.erb

```diff
@@ -15,7 +15,7 @@
         </div>
       </div>
       <div class="modal-footer">
-        <button type="button" class="btn btn-default" data-dismiss="modal"><%= _('ACK') %></button>
+        <button type="button" class="btn btn-default" data-dismiss="modal"><%= _('OK') %></button>
```

Comment:
Why not Overwrite too? Anyway, I created a conflict to force this show up and it's not working after I click on 'OK' (unrelated to this PR I know)